spec.md

```diff
@@ -51,7 +60,66 @@ for public re-use.
                 ├── 0         # Each multiscale level is stored as a separate Zarr array, as above, but only integer values
                 │   ...       # are supported.
                 └── n
+```
+
+### High-content screening
+
+The following layout should describe the hierarchy for a high-content screening
+dataset. There are exactly four levels of hierarchies above the images:
+
+- the top-level group defines a single plate and MUST implement the plate
+  specification defined below
+- the second group defines all acquisitions performed on the plate. If 
+  only one acquisition was performed, a single group must be used.
```

Comment:
See ongoing slack discussion. If this is a "must" then this conflicts with the "should" in the first statement, no?